Specs/addDefaultMatchers.js

```diff
@@ -241,6 +241,40 @@ define([
                 };
             },
 
+            toRenderPixelCount : function(util, customEqualityTesters) {
```

Comment:
Instead could this and below be a single function that just returns the number of the pixels? The comparison logic (equals and greater than) seems like it should belong in the individual tests.

Reply:
@pjcozzi suggested adding these as new matchers: https://github.com/AnalyticalGraphicsInc/cesium/pull/6069#discussion_r158381343

Reply:
I think the new matchers idea is good, but I think we can get away with just a single new matcher.